### DISCUSSION TITLE
Fatal Error 500 when saving E-Mail Account

### DISCUSSION THREAD

**Discussion by Laralaser** [rubenarslan/formr.org#569] [General]
Dear all,

It seems that there is an issue with sending emails. Since yesterday, emails are not being sent (showing "error_email_not_sent" in the activity logs with the explanation "SMTP connect() failed"). When trying to test the email account by saving it again, a 500 fatal error appears. A few months ago, this exact mail setup worked fine, so I don't think it's due to incorrect SMTP configuration on my side.

The fatal error message is:
2024-09-06 11:42:05
An unexpected error occurred. CODE HANDSOME-HEDGEHOG-7035.

Is anyone else experiencing this problem? Thanks in advance for any help or attention to this matter.
Also, thank you very much, Ruben and Cyril, for providing formr and for all the hard work and passion you put into this project!

Best regards,
Lara

**Comment by cyriltata** [comment]
Dear Lara

Sorry for the late reply. This bug has been addressed.

best
Cyril

**Reply by Laralaser** [reply]
Thank you so much, yes it works!!
Best
Lara
